Products/CMFPlone/browser/templates/test_rendering.pt

```diff
@@ -457,7 +482,7 @@
           <span class="optionsToggle">
             <input type="checkbox" checked="checked" class="noborder" id="pt_toggle" value="#" name="pt_toggle"
               onclick="javascript:toggleSelect(this, 'portal_type:list', true)"
-              src="http://localhost:8080/Plone/select_all_icon.png">
+              src="http://localhost:8080/Plone/select_all_icons.png">
```

Comment:
${context/portal_url}
```suggestion
              src="${context/portal_url}/select_all_icons.png">
```

Reply:
The whole section (everything below examples of how to use new icons view) is commented out. This is work in progress an should be done/cleaned up when most of the templates/sections in Plone are done.